snmp/README.md

```diff
@@ -4,211 +4,102 @@
+| `discovery_allowed_failures` | The number of times a discovered host can fail before being removed from the list of discovered devices.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `bulk_threshold`             | The number of symbols in a table that triggers a BULK request. This paramater is only relevant for SNMPv > 1.                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `tags`                       | A list of global tags to add to all SNMP metrics. Read more about [tagging in Datadog][6].                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 
-**Note**: See the [SNMP Library reference][4] for all configuration options.
+##### Sample config
 
 ```yaml
 init_config:
-  ## @param mibs_folder - string - required
-  # mibs_folder: <PATH_TO_ADDITIONAL_MIBS>
-
-  ## @param global_metrics - list of elements - (required if none at the instance level)
-  # global_metrics:
-  ## - MIB: <MIB_NAME>
-  ##   symbol: <SYMBOL>
+  @param profiles - object - optional
+  profiles:
+    f5-big-ip:
+      definition_file: f5-big-ip.yaml
+    router:
+      definition_file: generic-router.yaml
 
 instances:
-    ## @param ip_address - string - required
-  - ip_address: localhost
-
-    ## @param port - integer - required
-    port: 161
+   ## @param network_address - string - optional
+   network_address: "<NETWORK_ADDRESS>"
```

Comment:
Isn't this missing the leading `-` used to separate instances in this array?

```yaml
instances:
  - network_address: ...
    port: 161
    ...
```

Reply:
Does each entry here need a `-`? Or just `network_address`?

Reply:
Updated for `-` before `network_address` in most recent commit. Definitely let me know if I'm missing any more `-`

Reply:
`-` is used to delimit the configuration for a given instance (which represents a device, or a subnet to be discovered). AFAIU the example config describes configuration for a single auto-discovered subnet, so we should be good now, though I have pending comments on formatting. :-)